src/routes/startups/faq.svelte

```diff
@@ -93,7 +93,7 @@
                             class="flex w-full items-center justify-between gap-2.5 py-6 text-left"
                             use:melt={$trigger(`${index}`)}
                         >
-                            <span class="text-label text-primary">
+                            <span class="text-label font-aeonik-pro text-primary">
```

Comment:
can we avoid hard coding a font?

Reply:
Not with Tailwind's semantics, no. The idea is for it to be as modular as possible in those ways, and just be a combo of the utility classes.